custom_components/tuya_local/devices/point_panel_heater.yaml

```diff
@@ -0,0 +1,51 @@
+name: Point Panel Heater
+products:
+  - id: nqj51hxb2tx5kklv
+    name: Point Panel Heater POPANW600
+primary_entity:
+  entity: climate
+  translation_only_key: heater
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+        - dps_val: false
+          value: "off"
+    - id: 2
+      name: temperature
+      type: integer
+      unit: C
+      mapping:
+        - scale: 0.1
```

Comment:
This will result in a range of 50 to 500 degrees, with 10 degree steps.  Is it what you intended?

More likely the scale should be 10, and the range min: 50, max: 500